service/chain/protocol_state_test.go

```diff
@@ -0,0 +1,142 @@
+package chain_test
+
+import (
+	"math"
+	"testing"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/storage/badger/operation"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/awfm9/flow-dps/service/chain"
+)
+
+const (
+	testHeight               = 42
+	testBlockID              = "56faa737ff2b9242d1b0b7015b868aa376a623e76a65b64e8e28c7edd407113f"
+	testSeal                 = "7cef58c6e60f4c2d23cd6b7991a68297e1b6bcd95d99ad05feb0492db53dfd3c"
+	testCommit               = "04bce5f43dc990800869fa05da95e92f4bac4b33a7579fcd778efc7b76a884f2"
+	testChainID flow.ChainID = "flow-testnet"
+)
+
+func inMemoryDb(t *testing.T) *chain.ProtocolState {
```

Comment:
In Go, abbreviations are usually all upper case afaik: `inMemoryDB`.

Reply:
I would also argue that if we want to directly return the `*chain.ProtocolState` here, it should be `inMemoryChain`.